.travis.yml

```diff
@@ -1,8 +1,9 @@
 sudo: required
 dist: xenial # ubuntu 16.04 distro
 script:
+  - make deps
   # runs  only when PR is merged
-  - make versionmaker
+  - author=`git log -1 --pretty=format:'%an'` && if [[ "$author"  != "Travis CI" ]]; then make versionmaker; fi
 after_success: 
    # runs only when PR is merged and push the version file to master version
```

Comment:
Verify the comment once some typo is there

Reply:
I think this line should be `runs only when PR is merged and push the version file to master branch`